microservices/validateApi/validator/validator.py

```diff
@@ -48,35 +48,81 @@ def validate(rule, resultObj):
 
 
 def read_file_and_evaluate(source, result):
-    file_resp, file_attributes = read_file(result['file_id'], not(source.find('${file.content}') == -1))
+    _, file_attributes = read_file(
+        result['file_id'], '${file.content}' in source)
     return evaluate_source(source, file_attributes)
 
 
 def evaluate_source(source, file_attributes):
-    munchified_attributes = munchify(file_attributes)
-
-    source = source.replace("${file.", "{0.")
-
+    PREFIX = "file."
     result = False
     message = ""
 
+    # def format_fn(val):
+    #     """
+    #     This function munchifies the file attributes and formats string val
+    #     :param val: A template string
+    #     :return: An interpolated string
+    #     """
+    #     val_temp = val.replace("${file.", "{0.")
+    #     return val_temp.format(munchify(file_attributes))
+
     try:
-        source = source.format(munchified_attributes)
-        old_stdout = sys.stdout
-        redirected_stdout = sys.stdout = StringIO()
-        exec(source)
-        sys.stdout = old_stdout
-        execOutput = redirected_stdout.getvalue().lower()
-        print(execOutput)
-        result = execOutput in ("yes", "true", "t", "1", "yes\n", "true\n", "t\n", "1\n")
+        munch_attr = munchify(file_attributes)
+
```

Comment:
Add comments for this section to make it more clear what it's doing at a glance.